mappings/net/minecraft/SharedConstants.mapping

```diff
@@ -109,9 +111,39 @@ CLASS net/minecraft/unmapped/C_otltijcb net/minecraft/SharedConstants
 		COMMENT else {@code (1 << {@link #SNAPSHOT_PROTOCOL_VERSION_EXTRA_BITS}) + {@link #SNAPSHOT_PROTOCOL_VERSION}}}
 	METHOD m_ltryxeto setGameVersion (Lnet/minecraft/unmapped/C_xyxorzsk;)V
 		ARG 0 gameVersion
+	METHOD m_mnjwgyqe stripInvalidCharacters (Ljava/lang/String;)Ljava/lang/String;
+		COMMENT {@return {@code text} with all {@linkplain #isValidChar invalid characters} stripped}
+		COMMENT <p>
+		COMMENT LF (linebreak; U+000A) is an invalid character and therefore stripped.
+		COMMENT Use {@linkplain #stripInvalidCharacters(String, boolean)} to keep linebreaks.
+		COMMENT
+		COMMENT @see #isValidChar(char)
+		COMMENT @see #stripInvalidCharacters(String, boolean)
+		ARG 0 text
 	METHOD m_pfbdxkbd isValidChar (C)Z
+		COMMENT {@return {@code true} if the character is not
+		COMMENT {@linkplain net.minecraft.util.Formatting#FORMATTING_CODE_PREFIX the formatting code prefix}
+		COMMENT (&bsol;u00a7),
+		COMMENT C0 control code (&bsol;u0000 to &bsol;u001f) or delete (&bsol;u007f),
+		COMMENT or {@code false} otherwise}
+		COMMENT
+		COMMENT @apiNote This method is used to determine if the server should
+		COMMENT accept a chat message sent from client.
+		COMMENT @see net.minecraft.server.network.ServerPlayNetworkHandler#hasInvalidCharacters(String)
 		ARG 0 chr
 	METHOD m_redqmfkb isVoidTerrain (Lnet/minecraft/unmapped/C_ynrszrtu;)Z
 		ARG 0 chunkPos
 	METHOD m_rugsknyf getGameVersion ()Lnet/minecraft/unmapped/C_xyxorzsk;
 	METHOD m_shiahrfj enableDataFixerOptimizations ()V
+	METHOD m_xcyumsii stripInvalidCharacters (Ljava/lang/String;Z)Ljava/lang/String;
+		COMMENT {@return {@code text} with {@linkplain #isValidChar invalid characters} stripped}
+		COMMENT <p>
+		COMMENT LF (linebreak; U+000A) may or may not be stripped depending on the passed
+		COMMENT {@code allowLinebreaks} value.
+		COMMENT
+		COMMENT @see #isValidChar(char)
+		COMMENT @see #stripInvalidCharacters(String)
+		ARG 0 text
+			COMMENT the text to filter
+		ARG 1 allowLineBreaks
+			COMMENT {@code true} if LF (linbreak; U+000A) are allowed, or {@code false} otherwise
```

Comment:
Typo of `linebreak` here. How about this?
```suggestion
			COMMENT {@code true} if LF (linebreak; U+000A) characters should be kept, or {@code false} otherwise
```